roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
-  - pip
+  - role: python-app
+    name: zuul
+    python_pipdeps:
+      - name: jenkins-job-builder
+        version: '1.6.1'
+      - name: pyzmq
+      - name: ansible
+      - name: statsd
+    git_repo: "{{ zuul_git_repo_url }}"
+    git_version: "{{ zuul_git_branch }}"
+    python_notify: Restart zuul
```

Comment:
So when running it (with -vv) i was seeing output such as: 

```
TASK [python-app : Checkout source] ********************************************
task path: /home/jamie/work/bonny/hoist/roles/python-app/tasks/main.yml:27
NOTIFIED HANDLER Restart zuul
NOTIFIED HANDLER zuul : check zuul-server is running
changed: [jamielennox-playpen] => {"after": "bd8c1adad09f2ea80384d57ac2f873b4ecbf17ba", "before": null, "changed": true, "warnings": []}
```

Which is why i believe this was working, but i had not done an explicit test with a handler that was not being triggered otherwise. I will check and report.